kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/CSVSchemaCommand.java

```diff
@@ -116,11 +123,19 @@ public int run() throws IOException {
     if (requiredFields != null) {
       required = ImmutableSet.copyOf(requiredFields);
     }
+    
+    HashMap<String,String> types = new HashMap<String,String>();
```

Comment:
You can avoid parsing by using [dynamic parameters|http://jcommander.org/#DynamicParameters].